### PR TITLE
docs: troubleshoot, project grant missing

### PR DESCRIPTION
**Pull request by hifabienne**
Add troubleshooting page for error message "Login not possible. The organization of the user must be granted to the project"

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/support/troubleshooting.mdx [hunk 1]
```diff
@@ -73,3 +73,40 @@ The WebFinger requirement and setup is a step a user has to take outside of thei
 On their custom domain, e.g example.com, users need to host a WebFinger endpoint at https://example.com/.well-known/webfinger. When queried, this endpoint returns a JSON response detailing the issuer. Users would need to host the endpoint with the link to the ZITADEL issuer. Tailscale only looks up this endpoint once when a user signs up, and will only look up this endpoint again if the user needs to make a configuration change to their identity provider.
 
 The requirements and a set up guide is detailed in the [Tailscale documentation](https://tailscale.com/kb/1240/sso-custom-oidc/).
+
+## Login not possible. The organization of the user must be granted to the project
+
+![Organization must be granted Error](/img/support/login_not_possible_error.png)
+
+ZITADEL does is not only capable of handling authentication but also authorization.
```
Comment (mffap) [inline docs/docs/support/troubleshooting.mdx:81]:
```suggestion
ZITADEL is not only capable of handling authentication but also authorization.
```

File: docs/docs/support/troubleshooting.mdx [hunk 2]
```diff
@@ -73,3 +73,40 @@ The WebFinger requirement and setup is a step a user has to take outside of thei
...
+This error message tells you, that a permission is missing for the organization of the user.
```
Comment (mffap) [inline docs/docs/support/troubleshooting.mdx:82]:
```suggestion
The error message "Login not possible. The organization of the user must be granted to the project." tells you, that a permission is missing for the organization of the user. 
```

Comment (mffap) [reply docs/docs/support/troubleshooting.mdx:82]: Which user? I assume the user trying to login to the project?

File: docs/docs/support/troubleshooting.mdx [hunk 3]
```diff
@@ -73,3 +73,40 @@ The WebFinger requirement and setup is a step a user has to take outside of thei
...
+
+You do have two organizations, an owner (Organization A) and a customer (Organization B).
+The Organization A owns a Project, and has to grant it to Organization B, so users are allowed to authenticate.
+The error message tells that the permission is required, but the project is not granted to Organization B.
```
Comment (mffap) [inline docs/docs/support/troubleshooting.mdx:86]:
```suggestion
The error message is shown to users of Organization B that the permission is required, but the project is not granted to Organization B.
```

File: docs/docs/support/troubleshooting.mdx (hunk 2 above)
Comment (mffap) [reply docs/docs/support/troubleshooting.mdx:82]: Shouldn't we say "project grant" is missing instead of "permission"?

**Review by mffap — CHANGES_REQUESTED** [review]

File: docs/docs/support/troubleshooting.mdx (hunk 2 above)
Comment (hifabienne) [reply docs/docs/support/troubleshooting.mdx]: changed

**Review by mffap — APPROVED** [review]

**Pull request merged**